core/trino-main/src/main/java/io/trino/operator/DynamicFilterSourceOperator.java

```diff
@@ -418,7 +418,7 @@ private ChannelFilter(
                     blockTypeOperators.getEqualOperator(type),
                     blockTypeOperators.getHashCodeOperator(type),
                     blockBuilder,
-                    Math.min(maxDistinctValues, 8192) * 2,
```

Comment:
unrelated change?

Reply:
Once we switch to higher default, the sizing factor on maxDistinctValues doesn't need to be as aggressive.
The previous value of 8192 also turned out to be good in microbenchmarking but a bit over eager in using memory for actual queries. Not big enough to notice any change in peak memory usage but I decided to make it more conservative anyway.